### DISCUSSION TITLE
Lower case all fields in Sigma Rules

### DISCUSSION THREAD

**Discussion by nNipsx-Sec** [SigmaHQ/pySigma#54] [Ideas]
When created fields mapping in pipelines, i think make all fields in sigma rules import to lower case to easier to created mapping

**Comment by frack113** [comment]
Hello,
I don't think it is a good idea.
for example
`OriginalFileName` is use by sysmon.
`OriginalFilename` is the field use by microsoft ETW when rename a file.
`Filename` and `FileName` are not the same.

You have to check the behavior of the backend

**Reply by nNipsx-Sec** [reply]
Got it. Thanks for reccommend 😀
